python/python/glide/async_commands/cluster_commands.py

```diff
@@ -46,7 +56,7 @@ async def info(
         self,
         sections: Optional[List[InfoSection]] = None,
         route: Optional[Route] = None,
-    ) -> TClusterResponse[str]:
+    ) -> TClusterResponse[bytes]:
```

Comment:
I don't think that `info` can ever contain binary strings, so you can keep response type unchanged.

Reply:
Based on many discussions, we decided to be consistent with the protocol (Resp3) by returning bytes for anything that returns BulkStrings